docs/pages/reference/networking.mdx

```diff
@@ -124,45 +125,53 @@ numbers for each service.
 
 </Admonition>
 
-### Ports with TLS routing
+#### Ports with TLS routing
 
-TLS routing is enabled by default. In this mode, all connections to a Teleport service (e.g., the Teleport
-SSH Service or Kubernetes) are routed through the Proxy Service's public web address.
+TLS routing is enabled by default. In this mode, all connections to a Teleport
+service (e.g., the Teleport SSH Service or Kubernetes) are routed through the
+Proxy Service's public web address.
 
 Read more in our [TLS Routing](../architecture/tls-routing.mdx) guide.
 
-| Port | Service | Description |
+| Port | Downstream Service | Description |
 | - | - | - |
-| 443 | Proxy | In TLS Routing mode, the Proxy handles all protocols, including Web UI, HTTPS, Kubernetes, SSH, and all databases on a single port. |
-| 3021 | Proxy | Port used by Teleport Proxy Service instances to dial agents in Proxy Peering mode. |
-| 3022 | Node | SSH port. This is Teleport's equivalent of port `#22` for SSH. Only used when Teleport Node is replacing SSH.|
-| 3025 | Auth | TLS port used by the Auth Service to serve its API to other Nodes in a cluster.|
-| 3028 | Desktop | When using Desktop Service `windows_desktop_service.listen_addr` |
+| 443 | Proxy Service | In TLS Routing mode, the Proxy handles all protocols, including Web UI, HTTPS, Kubernetes, SSH, and all databases on a single port. |
+| 3021 | Proxy Service | Port used by Teleport Proxy Service instances to dial agents in Proxy Peering mode. |
+| 3022 | SSH Service | SSH port. This is Teleport's equivalent of port `#22` for SSH. Only used when Teleport agent is replacing SSH.|
```

Comment:
`3022` is the port exposed on an SSH agent, if the Proxy isn't running an SSH agent, this port won't be in use. So it seems a little weird for it to be under "Proxy Service ports"

Reply:
Ah, must have forgotten about/missed this one. Is the SSH Service the only agent service that requires an open port?

Reply:
It doesn't /require/ an open port to the internet due to the reverse tunnelling to the Proxy, but, most of the agent services do open ports to enable direct access. Maybe it's worth breaking all the agent ports out into their own section ?

Reply:
That's looking better - I think some of those agent ports could then be removed from the Proxy service ports tables. I'd only really expect to see ports under the Proxy service ports header that are configured in the `proxy_service` section of the config. If it's configured in another section, then I'd probably expect it to show under the Agent ports header.